pkgs/development/compilers/zig/default.nix

```diff
@@ -1,30 +1,23 @@
-{ stdenv, fetchFromGitHub, cmake, llvmPackages, libxml2 }:
+{ stdenv, fetchFromGitHub, cmake, llvmPackages, libxml2, zlib }:
 
 stdenv.mkDerivation rec {
-  version = "0.2.0";
+  version = "0.3.0";
   name = "zig-${version}";
 
   src = fetchFromGitHub {
-    owner = "zig-lang";
+    owner = "ziglang";
     repo = "zig";
     rev = "${version}";
-    sha256 = "0lym28z9mj6hfiq78x1fsd8y89h8xyfc1jgqyazi1g9r72427n07";
+    sha256 = "089ywagxjjh7gxv8h8yg7jpmryzjf7n4m5irhdkhp2966d03kyxm";
   };
 
   nativeBuildInputs = [ cmake ];
-  buildInputs = [ llvmPackages.clang-unwrapped llvmPackages.llvm libxml2 ];
+  buildInputs = [ llvmPackages.clang-unwrapped llvmPackages.llvm libxml2 zlib ];
 
   cmakeFlags = [
-    "-DZIG_LIBC_INCLUDE_DIR=${stdenv.cc.libc_dev}/include"
-    "-DZIG_LIBC_LIB_DIR=${stdenv.cc.libc}/lib"
     "-DCMAKE_BUILD_TYPE=Release"
-    "-DZIG_EACH_LIB_RPATH=On"
   ];
 
-  preConfigure = ''
-    cmakeFlags="$cmakeFlags -DZIG_LIBC_STATIC_LIB_DIR=$(dirname $(cc -print-file-name=crtbegin.o)) -DZIG_DYNAMIC_LINKER=$(cc -print-file-name=ld-linux-x86-64.so.2)"
-  '';
-
```

Comment:
These are no longer needed? :grin:

Reply:
Correct! This is a happy side-effect of Zig's static Linux binaries (which even work on NixOS). libc is detected at runtime.